docs/SOURCES.md

```diff
@@ -1,255 +1,972 @@
+| 🇫🇷 France               | [Météo-France](#météo-france)                      | 14 days  |
+| 🇩🇪 Germany              | [Bright Sky](#bright-sky)                          | 10 days  |
+| 🇬🇮 Gibraltar            | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇬🇱 Greenland            | [DMI](#danmarks-meteorologiske-institut)           | 10 days  |
+| 🇬🇺 Guam                 | [NWS](#national-weather-service)                   | 7 days   |
+| 🇬🇬 Guernsey             | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇭🇰 Hong Kong            | [HKO](#hong-kong-observatory)                      | 10 days  |
+| 🇮🇳 India                | [IMD](#india-meteorological-department)            | 10 days  |
+| 🇮🇩 Indonesia            | [BMKG](#bmkg)                                      | 9 days   |
+| 🇮🇪 Ireland              | [MET Éireann](#met-éireann)                        | 7 days   |
+| 🇮🇲 Isle of Man          | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇮🇱 Israel               | [IMS](#israel-meteorological-service)              | 6 days   |
+| 🇮🇹 Italy                | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+| 🇯🇪 Jersey               | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇱🇺 Luxembourg           | [MeteoLux](#meteolux)                              | 5 days   |
+| 🇲🇴 Macao                | [SMG](#serviços-meteorológicos-e-geofísicos)       | 7 days   |
+| 🇲🇨 Monaco               | [Météo-France](#météo-france)                      | 14 days  |
+| 🇲🇳 Mongolia             | [NAMEM](#namem)                                    | 5 days   |
+| 🇲🇵 Northern Mariana Is. | [NWS](#national-weather-service)                   | 7 days   |
+| 🇳🇴 Norway               | [MET Norway](#met-norway)                          | ~10 days |
+| 🇵🇭 Philippines          | [PAGASA](#pagasa)                                  | 5 days   |
+| 🇵🇹 Portugal             | [IPMA](#instituto-português-do-mar-e-da-atmosfera) | 10 days  |
+| 🇵🇷 Puerto Rico          | [NWS](#national-weather-service)                   | 7 days   |
+| 🇸🇲 San Marino           | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+| 🇸🇯 Svalbard & Jan Mayen | [MET Norway](#met-norway)                          | ~10 days |
+| 🇸🇪 Sweden               | [SMHI](#smhi)                                      | 15 days  |
+| 🇹🇼 Taiwan               | [CWA](#central-weather-administration) 🔐          | 7 days   |
+| 🇹🇷 Türkiye              | [MGM](#meteoroloji-genel-müdürlüğü)                | 5 days   |
+| 🇬🇧 United Kingdom       | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇺🇸 United States        | [NWS](#national-weather-service)                   | 7 days   |
+| 🇻🇮 U.S. Virgin Is.      | [NWS](#national-weather-service)                   | 7 days   |
+| 🇻🇦 Vatican City         | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+
```

Comment:
I think you can list the 3 specific sources in this summary (or index of contents) so that we know it exists

Reply:
Did you mean the secondary-only sources?

Reply:
Yes

Reply:
Added to the latest revision.